internal/cmd/egctl/testdata/translate/out/default-resources.all.yaml

```diff
@@ -322,6 +322,7 @@ grpcRoutes:
       - method:
           method: DoThing
           service: com.example.Things
+      sessionPersistence: null
```

Comment:
should be fixed in v1.1
https://github.com/kubernetes-sigs/gateway-api/pull/3059